### ISSUE TITLE
Error when defining a custom location of configfile and analyzing files inside same location

### ISSUE THREAD

**Issue by gian1200** [microsoft/vscode-eslint#1946]
Continuation of https://github.com/microsoft/vscode-eslint/issues/1932

Issue happens when setting config file in a custom location (e.g. `<project-path>\.config\eslint.config.mjs`)

> I think I've pin point the issue. It happens only for files inside `.config` folder. For any file outside, `VSCode-ESLint` works fine.
> 
> Steps to reproduce:
> 1. git clone [reproducer](https://github.com/gian1200/reproduce-vscode-eslint-issue)
> 2. `cd reproduce-vscode-eslint-issue` and `npm install` inside the root folder
> 3. `code .` --> open the project on VSCode and make sure vscode-eslint is enabled (wait for everything to load). If not loaded, a VSCode reload might be necessary.
> 4. Open files for testing:
>     - `src/dummy.mjs`: It is flagged correctly by VSCode-ESLint
>     - `.config/anotherDummyFile.mjs`: **This file is not flagged at all (see Output for error)**
> 5. Execute `npm run eslint`: both files are correctly flagged

**Comment by dbaeumer** [comment]
The problem is that we determine the working directory based on the config file location. As a result eslint looks for the config file in .config underneath .config.

You can use `"eslint.workingDirectories": [.]` to fix the working directory to the root.

**Comment by gian1200** [comment]
Awesome, it worked!

Just one minor comment for anyone reading this in the future.

1. The file to edit is `.vscode/settings.json`
2. Add quotes around the dot. See below:

```json
"eslint.workingDirectories": ["."]
```

Thanks!
